include/deal.II/meshworker/mesh_loop.h

```diff
@@ -393,6 +393,71 @@ namespace MeshWorker
                     chunk_size);
   }
 
+  /**
+   * Same as the function above, but for iterator ranges (and, therefore,
+   * filtered iterators).
+   */
+  template <class CellIteratorRangeType, class ScratchData, class CopyData>
+  void
+  mesh_loop(
+    CellIteratorRangeType iterator_range,
+    const typename identity<
+      std::function<void(const typename CellIteratorRangeType::iterator &,
```

Comment:
here and for the next argument, do you really want to apply the identity to the whole thing, or just to the inner `CellIteratorRange` argument?